db-facilities/facdb/sql/dsny_electronicsdrop.sql

```diff
@@ -2,13 +2,13 @@ DROP TABLE IF EXISTS _dsny_electronicsdrop;
 SELECT uid,
     source,
     CONCAT(dropoff_sitename, ' Electronics Drop-off Site') as facname,
-    number as addressnum,
-    street as streetname,
+    NULL as addressnum,
+    NULL as streetname,
     address,
     NULL as city,
     zipcode,
-    NULL as boro,
-    LEFT(borocd, 1) as borocode,
+    borough as boro,
+    NULL as borocode,
```

Comment:
Glanced at the dataset, this one is a bit of a doozy in terms of changes it seems.

LEFT(bbl, 1) should likely be a more steady way to get borocode, I don't think that should be dropping from the dataset. And if it does, it's its own column so it will present more issues

Reply:
Thanks! Will give that a shot.